dataquality/dq_auto/schema.py

```diff
@@ -46,6 +48,9 @@ class BaseAutoDatasetConfig:
     # Column names
     input_col: str = "text"
     target_col: str = "label"
+    # Dataset input / output formatter
+    max_train_size: Optional[int] = None
+    formatter: BaseFormatter = DefaultFormatter()
```

Comment:
This is a more idiomatic way of initializing for a dataclass:

```suggestion
from dataclasses import field
...
    formatter: BaseFormatter = field(default_factory=DefaultFormatter)
```